pjsip/src/pjsua2/account.cpp

```diff
@@ -595,6 +595,9 @@ void AccountConfig::toPj(pjsua_acc_config &ret) const
 	dst->ext.aka.k	= str2Pj(src.akaK);
 	dst->ext.aka.op	= str2Pj(src.akaOp);
 	dst->ext.aka.amf= str2Pj(src.akaAmf);
+#if PJSIP_HAS_DIGEST_AKA_AUTH
+	dst->ext.aka.cb = &pjsip_auth_create_aka_response;
```

Comment:
toPj() is supposed to just do conversion, it shouldn't do a new assignment.

You should add a new field akaCb, then here in toPj() will be just:
```
dst->ext.aka.cb = src.akaCb;
```